helpers/wrapper/contracts/Wrapper.sol

```diff
@@ -62,30 +62,30 @@ contract Wrapper {
 
   /**
     * @notice Send an Order
-    * @dev Taker must authorize this contract on the swapContract
-    * @dev Taker must approve this contract on the wethContract
+    * @dev Sender must authorize this contract on the swapContract
+    * @dev Sender must approve this contract on the wethContract
     * @param _order Types.Order
     */
   function swap(
     Types.Order calldata _order
   ) external payable {
 
-    // Ensure message sender is taker wallet.
-    require(_order.taker.wallet == msg.sender,
+    // Ensure msg.sender is sender wallet.
+    require(_order.sender.wallet == msg.sender,
       "SENDER_MUST_BE_TAKER");
```

Comment:
`MESSAGE_SENDER_MUST_BE_ORDER_SENDER` ?

Reply:
that would be too long.... MSGSENDER_MUST_BE_SENDER